src/main/java/org/spongepowered/api/text/title/Title.java

```diff
@@ -204,4 +207,390 @@ public String toString() {
+                    && this.reset == that.reset;
+        }
+
+        @Override
+        public int hashCode() {
+            return Objects.hashCode(this.title, this.subtitle, this.fadeIn, this.stay, this.fadeOut, this.clear, this.reset);
+        }
+
+        @Override
+        public String toString() {
+            return Objects.toStringHelper(this)
+                    .add("title", this.title)
+                    .add("subtitle", this.subtitle)
+                    .add("fadeIn", this.fadeIn)
+                    .add("stay", this.stay)
+                    .add("fadeOut", this.fadeOut)
+                    .add("clear", this.clear)
+                    .add("reset", this.reset)
+                    .toString();
+        }
+
+    }
+
+    /**
+     * Returns a {@link Title} that will simply do nothing when it is sent to
+     * the client.
+     *
+     * @return An empty title instance
+     */
+    public static Title of() {
+        return EMPTY;
+    }
+
+    /**
+     * Returns a {@link Title} that will display the given main title on the
+     * player's screen.
+     *
+     * @param title The title to display
+     * @return The created title
+     */
+    public static Title of(Text title) {
+        return builder().title(title).build();
+    }
+
+    /**
+     * Returns a {@link Title} that will display the given main and subtitle on
+     * the player's screen.
+     *
+     * @param title The title to display
+     * @param subtitle The subtitle to display
+     * @return The created title
+     */
+    public static Title of(Text title, Text subtitle) {
+        return builder().title(title).subtitle(subtitle).build();
+    }
+
+    /**
+     * Returns a {@link Title} that will clear the currently displayed
+     * {@link Title} from the player's screen.
+     *
+     * @return A title configuration that will clear
+     */
+    public static Title clear() {
+        return CLEAR;
+    }
+
+    /**
+     * Returns a {@link Title} that will reset the current title back to default
+     * values on the client.
+     *
+     * @return A title configuration that will reset
+     */
+    public static Title reset() {
+        return RESET;
+    }
+
+    /**
+     * Creates a new {@link Title} configuration builder that will reset the
+     * currently displayed Title on the client before displaying the new
+     * configured one.
+     *
+     * @return A new {@link Builder}
+     * @see #update
+     */
+    public static Builder builder() {
+        return update().reset();
+    }
+
+    /**
+     * Creates a new empty {@link Title} configuration builder. Unlike
+     * {@link #builder} this won't reset the current Title on the client before
+     * displaying the current one. This has less use cases but should be used if
+     * just the previously sent Title should be updated.
+     *
+     * @return A new {@link Builder}
+     * @see #builder
+     */
+    public static Builder update() {
```

Comment:
This method is... strange. I'm not sure if it is the description or the name (or both) that will cause confusion.